app/assets/stylesheets/container_providers_dashboard.css

```diff
@@ -46,7 +46,7 @@
 
 
 /* Remove after next pf release */
-.icon-atomic:before {
+.pficon-atomic:before {
```

Comment:
> how is this change related to this PR?

Im auto-generating the icon-class field so I made the icon class for atomic to adhere to the rest.
(which are ```pficon-kubernetes``` and ```pficon-openshift```) 
when atomic will be added to PF Ill just be able to remove the ```.pficon-atomic``` class from ```container_providers_dashboard.css```